code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm

```diff
@@ -359,4 +359,387 @@ Difficulty: Very Hard
 /obj/machinery/smartfridge/black_box/default_deconstruction_crowbar()
 	return
 
+///Anomolous Crystal///
+
+/obj/machinery/anomalous_crystal
+	name = "anomalous crystal"
+	desc = "A strange chunk of crystal, being in the presence of it fills you with equal parts excitement and dread."
+	icon = 'icons/obj/lavaland/artefacts.dmi'
+	icon_state = "anomaly_crystal"
+	luminosity = 8
+	burn_state = LAVA_PROOF
+	use_power = 0
+	density = 1
+	languages_spoken = ALL
+	languages_understood = ALL
+	flags = HEAR
+	var/activation_method = "touch"
+	var/activation_damage_type = null
+	var/last_use_timer = 0
+	var/cooldown_add = 30
+	var/list/affected_targets = list()
+	var/activation_sound = 'sound/effects/break_stone.ogg'
+
+/obj/machinery/anomalous_crystal/New()
+	activation_method = pick("touch","laser","bullet","energy","magic","mob_bump","heat","weapon","speech")
+	..()
+
+/obj/machinery/anomalous_crystal/Hear(message, atom/movable/speaker, message_langs, raw_message, radio_freq, spans)
+	..()
+	if(isliving(speaker))
```

Comment:
I was under the impression that because things like intercoms can "speak" it would be safer to check if it's at least a living mob.